packages/cli/docs-resolver/src/ApiReferenceNodeConverter.ts

```diff
@@ -370,13 +371,23 @@ export class ApiReferenceNodeConverter {
 
     #convertEndpoint(
         endpointItem: docsYml.ParsedApiReferenceLayoutItem.Endpoint,
-        apiDefinitionPackageId: string | undefined,
-        parentSlug: FernNavigation.SlugGenerator,
+        apiDefinitionPackageIdRaw: string | undefined,
+        parentSlug: FernNavigation.V1.SlugGenerator,
         idgen: NodeIdGenerator
-    ): FernNavigation.ApiPackageChild | undefined {
+    ): FernNavigation.V1.ApiPackageChild | undefined {
+        if (!apiDefinitionPackageIdRaw) {
+            this.taskContext.logger.error(
+                "No ApiDefinition ID supplied in the API reference layout: ",
+                endpointItem.endpoint
+            );
```

Comment:
this error message is going to be put in front of a user fyi so we probably want to make it far more friendly and even better `fern check` it